### PR TITLE
Change commonmarker to qiita marker

### PR DESCRIPTION
**Pull request by ohakutsu**
## What

I did the following to change this library name to **Qiita Marker**.

- Add rake task to rename `commonmarker` to `qiita_marker`
- Add rake task to move submodules of `ext/commonmarker/cmark-upstream` to `ext/qiita_marker/cmark-upstream`
- Rename `commonmarker` in paths and sources to `qiita_marker`
    - Run `bundle exec rake qiita_marker:rename_project`
- Move submodule of libcmark to `ext/qiita_marker`
    - Run `bundle exec rake qiita_marker:move_submodules`

### CODE REVIEW

**Review by atm-snag2 — APPROVED** [review]
LGTM

File: qiita_marker.gemspec [hunk 1]
```diff
@@ -2,28 +2,28 @@
...
 Gem::Specification.new do |s|
-  s.name = 'commonmarker'
-  s.version = CommonMarker::VERSION
+  s.name = 'qiita_marker'
+  s.version = QiitaMarker::VERSION
   s.summary = 'CommonMark parser and renderer. Written in C, wrapped in Ruby.'
   s.description = 'A fast, safe, extensible parser for CommonMark. This wraps the official libcmark library.'
   s.authors = ['Garen Torikian', 'Ashe Connor']
-  s.homepage = 'https://github.com/gjtorikian/commonmarker'
+  s.homepage = 'https://github.com/gjtorikian/qiita_marker'
```
Comment (kyntk) [inline qiita_marker.gemspec:13]: This url is wrong. The correct one is here.
https://github.com/increments/qiita_marker

Comment (ohakutsu) [reply qiita_marker.gemspec:13]: @kyntk 
This pull request only do the mechanical renaming.
I'll fix the `gemspec` in another pull request.

**Review by kyntk — APPROVED** [review]
LGTM!

**Pull request merged**